Sources/Foundation/NSSwiftRuntime.swift

```diff
@@ -14,22 +14,19 @@
 // This mimics the behavior of the swift sdk overlay on Darwin
 #if os(macOS) || os(iOS) || os(tvOS) || os(watchOS)
 @_exported import Darwin
-#elseif os(Linux) || os(Android) || CYGWIN || os(OpenBSD)
+#elseif os(Linux) || os(Android) || os(Cygwin) || os(OpenBSD)
 @_exported import Glibc
 #elseif os(WASI)
 @_exported import WASILibc
 #elseif os(Windows)
 @_exported import CRT
+@_exported import WinSDK.core
```

Comment:
See the discussion. Why are we increasing the exported surface?

Reply:
`WinSDK.core` should be regarded as the real system interface of Windows, far better and more native the broken POSIX support. We’re using its functionality almost everywhere now, and part of it is already in the interface.

Given that POSIX are exported in every other platform, I think re-exporting `WinSDK.core` is reasonable. If we don’t do this, we’ll have to add `import WinSDK` in several files (more harmful as we may accidentally export symbols from modules other than `core`), and will get a broken exported `WinSDK` interface which users can hardly make use of. On other platforms, all system functions are exported, and Windows shouldn’t be an exception.

Reply:
I'm still a bit confused on this part. Why is it the responsibility of Foundation to re-export the Windows SDK interface?

Reply:
I tried to eliminate `import WinSDK` to make sure that we don’t use APIs out of `core`, and I regarded as re-exporting `WinSDK.core` as reasonable because we’re re-exporting the whole of `Darwin` and `Glibc`. Now I reverted the change and added back all the missing imports.👀